components/style_derive/to_computed_value.rs

```diff
@@ -13,6 +13,29 @@ pub fn derive(input: DeriveInput) -> Tokens {
     let (impl_generics, ty_generics, mut where_clause, computed_value_type) =
         cg::fmap_trait_parts(&input, &trait_path, Ident::from("ComputedValue"));
 
+    if input.generics.params.is_empty() {
```

Comment:
Can you document in the general documentation for the derive stuff this? And that deriving `ToComputedValue` where `ComputedValue = Self` is bogus if you nest the type?